managed-boundary.tf

```diff
@@ -290,7 +290,7 @@ module "eks_brokerpak_policy" {
 
   name        = "eks_brokerpak_policy"
   path        = "/"
-  description = "Policy granting additional permissions needed by the EKS brokerpak"
+  description = "Policy granting additional permissions needed by the EKS brokerpak; most are for managing PVs"
```

Comment:
I'm conflicted about consolidating all of the policies required by the EKS brokerpak.  If we do need to figure out which permissions are for which, I suppose looking at the EKS brokerpak will do.  While it does simplify this object by putting everything together, it also makes it less readable..  If something breaks with IAM permissions, it might be tricky to track it down.. but that I think was an existing problem, so I think it's okay for now.

Reply:
I feel the same way. I spent some time trying to figure out how to add comments to policy JSON and I think there is something you can do with `sid` but I was not smart enough to figure it out last night. I'll try again today.

Reply:
Well, the `sid` is just a sort of [human-readable Identifier](https://docs.aws.amazon.com/IAM/latest/UserGuide/reference_policies_elements_sid.html) for that block of permissions.  I guess that would work..  We could break up the big policy in acceptable smaller blocks and label it with the `sid`.

Reply:
I like the idea of maybe another repo for just permissions like how we do for [this](https://github.com/gSA/datagov-iam).  But not sure if that's what you meant by "source it from another location".  I'm not familiar with any other external sources for IAM Policies that we would be able to control.

Reply:
If [the policy in the new IRSA module](https://github.com/terraform-aws-modules/terraform-aws-iam/blob/724651ec4d7e5ae12034ddc9e78a794d2a57eb9b/modules/iam-role-for-service-accounts-eks/policies.tf#L115-L308) matches what we need, we can switch to using that and also get tighter least-privilege management.

Reply:
(I'll investigate this later today.)